quickstarts/taskqueues-backup-images/README.md

```diff
@@ -0,0 +1,78 @@
+# Quickstart: Back up images using a Task Queue Function
```

Comment:
Do you think the README should discuss the values chosen for rate limits, backoff, etc?  

Not necessary afaik, but I feel like I'd like to discuss them in the doc/guide.

Reply:
Great suggestion! I'll try to expand on the task queue configuration on the README.md.